metrics/template.yaml

```diff
@@ -11,10 +11,10 @@ Resources:
       Description: A Lambda function that aggregates Lambda execution times by day and by runtime
       CodeUri: ./src/logsToMetrics/
       Handler: logsToMetrics::logsToMetrics.Function::FunctionHandler
-      Runtime: dotnet6
+      Runtime: dotnet8
       Architectures:
         - x86_64
-      MemorySize: 512
+      MemorySize: 2048
```

Comment:
smell test: why do you need 4x memory for .net 8?

Reply:
Slightly faster development and testing. This runs once a day for a few seconds, so the extra cost in compute is negligible. This also give more CPU, which is what I really want here.